src/__tests__/Cell.js

```diff
@@ -17,6 +17,6 @@ describe('Cell', () => {
       </Cell>
     );
 
-    expect(cell.find('.sticky-table-cell span')).to.have.length(2);
+    expect(cell.find(Cell).find('span')).to.have.length(2);
```

Comment:
@horacioh nevermind about tests - this was the only one I noticed, and it's clear the way you have it